### ISSUE TITLE
[TODO] 구현해야할 기능 리스트

### ISSUE THREAD

**Issue by nayohan** [boostcampaitech2/image-classification-level1-04#5]
[TODO}
- [ ] Weights & Biases 실험 로깅 기능 추가 
- [x] Augmentation 기능 (transforms, albumentaion, Oversampling, Undersapling) 추가
- [ ] Ray Tune 추가(? 확인 필요)
- [x] timm pretrained model 추가
- [x] Evaluation metric f1 추가
- [ ] Loss function 추가
